webapp/src/test/java/com/box/l10n/mojito/smartling/SmartlingClientTest.java

```diff
@@ -88,6 +89,48 @@ public void testGetSourceStringsStream() {
         });
     }
 
+    @Ignore
```

Comment:
Marking these tests as ignored to avoid delaying the test suite, but all tests were run using this code